internal/datacoord/garbage_collector.go

```diff
@@ -168,35 +277,40 @@ func (gc *garbageCollector) GetStatus() GcStatus {
 	}
 }
 
-func (gc *garbageCollector) Pause(ctx context.Context, pauseDuration time.Duration) error {
+func (gc *garbageCollector) Pause(ctx context.Context, collectionID int64, ticket string, pauseDuration time.Duration) error {
```

Comment:
Any compatibility issue? Since the API is changed. Saying during the roll upgrade

Reply:
nope, all internal function call